stable/yourls/templates/secrets.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+type: Opaque
+data:
+  {{ if .Values.yourlsPassword }}
+  yourls-password: {{ default "" .Values.yourlsPassword | b64enc | quote }}
+  {{ else }}
+  yourls-password: {{ randAlphaNum 10 | b64enc | quote }}
```

Comment:
@LeoColomb I do not like this pattern. 
On helm update, `yourls-password` would be changed, and there is no place where you could find previous one.

Reply:
@goruha I took the example here: 
https://github.com/helm/charts/blob/3b0a8f9bb14e18f25022a0a51e41fee0020c2cf4/stable/wordpress/templates/secrets.yaml#L15
Honestly, I don't know at all if this pattern is a bad practice, let me know.

Reply:
@LeoColomb yea it is. `randAlphaNum 10` would generate new value on each `helm install\update` so after the first `update` password in secrets would be different from ones stored in db.
So if you did not save password that was in secrets after install - you lost it.
Or you have any special script to sync them?

Reply:
Just do some default password like `yourlsPWD2019!HappyNewYear` :)

Reply:
You suggest a static password, right?
This would lead to a direct security weakness, it's even worst, if I'm right... 🤔 

And again, WordPress chart have the exact same behavior, I wonder if some reports have been made regarding always regenerated password?

To be honest I'm a bit confused on this point.
@darend What do you think about this?

Reply:
Both approaches have a weakness, however I see the use of a random password being the default convention for major charts:

Mysql: https://github.com/helm/charts/blob/b585508f338441c726cf042095553230e81ac0b7/stable/mysql/templates/secrets.yaml#L18-L22
Postgres: https://github.com/helm/charts/blob/b585508f338441c726cf042095553230e81ac0b7/stable/postgresql/templates/secrets.yaml#L13-L17

Mongo: https://github.com/helm/charts/blob/b585508f338441c726cf042095553230e81ac0b7/stable/mongodb/templates/secrets.yaml#L21-L25

Reply:
@goruha I would stick with @LeoColomb approach as it is the established pattern for other major charts